sdk/python/feast/infra/online_stores/contrib/hazelcast_online_store/README.md

```diff
@@ -0,0 +1,87 @@
+# Hazelcast Online Store
+
+This contribution makes it possible to use [Hazelcast](https://hazelcast.com/) as online store for Feast.
+
+Once the Hazelcast client configuration is given inside `feature_store.yaml` file, everything else
+is handled as with any other online store: schema creation, read/write from/to Hazelcast and remove operations.
+
+## Quick usage
+
+The following refers to the [Feast quickstart](https://docs.feast.dev/getting-started/quickstart) page. 
+Only the Step 2 is different from this tutorial since it requires you to configure your Hazelcast online store.
+
+### Creating the feature repository
+
+The easiest way to get started is to use the Feast CLI to initialize a new
+feature store. Once Feast is installed, the command
+
+```
+feast init FEATURE_STORE_NAME -t hazelcast
+```
+
+will interactively help you create the `feature_store.yaml` with the
+required configuration details to access your Hazelcast cluster.
+
+Alternatively, you can run `feast init -t FEATURE_STORE_NAME`, as described
+in the quickstart, and then manually edit the `online_store` section in
+the `feature_store.yaml` file as detailed below.
+
+The following steps (setup of feature definitions, deployment of the store,
+generation of training data, materialization, fetching of online/offline
+features) proceed exactly as in the general Feast quickstart instructions.
+
+#### Hazelcast setup
+
+In order to use [Hazelcast](https://hazelcast.com) as online store, you need to have a running Hazelcast cluster. 
+You can create a cluster using Hazelcast Viridian Serverless easily or deploy one on your local/remote machine. 
+See this [getting started](https://hazelcast.com/get-started/) page for more details.
+
+Hazelcast online store provides capability to connect TLS/SSL enabled cluster or Hazelcast Viridian Serverless cluster. 
+Following is an example to connect local cluster named "dev" running on port 5701 with SSL enabled.
+
+```yaml
+[...]
+online_store:
+    type: hazelcast
+    cluster_name: dev
+    cluster_members: ["localhost:5701"]
+    ssl_cafile_path: /path/to/ca/file
+    ssl_certfile_path: /path/to/cert/file
+    ssl_keyfile_path: /path/to/key/file
+    ssl_password: <YOUR_SSL_PASSWORD>
```

Comment:
nit, this should be read from the enrvironment instead of being hardcoded in the `feature_store.yaml` file. 

```suggestion
    ssl_password: ${SSL_PASSWORD} # This value will be read form the `SSL_PASSWORD` environment variable.
```

Reply:
That's a good idea, thanks! Replaced the SSL password with environment variable at https://github.com/feast-dev/feast/pull/3523/commits/dad8fc414d91205c63c1048d956cafe9ae316be6.